package/caffemodel-resnet50-intel-i8/.cm/meta.json

```diff
@@ -0,0 +1,85 @@
+{
+  "check_exit_status": "yes", 
+  "customize": {
+    "extra_dir": "",
+    "file_with_weights": "ResNet-50-model.caffemodel",
+    "file_mean_bin": "ResNet_mean.binaryproto",
+    "force_ask_path": "yes", 
+    "install_env": {
+      "DEPLOY_BATCH_SIZE": 1,
+      "DEPLOY_BLOB_NAME": "prob",
+      "DEPLOY_ITERATIONS": 4,
+      "TRAIN_VAL_BATCH_SIZE": 50,
+      "TRAIN_VAL_BLOB_NAME": "accuracy/top-1",
+      "TRAIN_VAL_ITERATIONS": 4,
+      "MODEL_IMAGE_SIZE": 224
+    }, 
+    "no_os_in_suggested_path": "yes", 
+    "no_ver_in_suggested_path": "no", 
+    "params": {
+      "val": {
+        "template": "train_val_quantized.prototxt",
+        "substitute":{
+          "val_batch_size": 50
+        },
+        "accuracy_layers": [ "accuracy/top-1", "accuracy/top-5", "fc1000" ]
```

Comment:
Should `fc1000` be here?

Reply:
Seems `prob` should be here, changed.
Actually I'm not sure for what it at all. I've taken it from alexnet package meta, it says
```
"accuracy_layers": [ "accuracy", "accuracy_top5", "loss" ]
```
last `loss` layer is SoftmaxWithLoss. For ResNet it is `prob`.